components/MainNav/MainNav.tsx

```diff
@@ -53,29 +48,31 @@ export const MainNav = ({
               <div className={styles.panelWrapper}>
                 <Grid sm={2} xxl={3} className={styles.panelOuterGrid}>
                   <div>
-                    <ul className={styles.linkList}>
-                      {featuredLinks.map((link) => (
-                        <li key={link._uid} className={styles.featuredItem}>
-                          <CtaLink
-                            href={link.href}
-                            color="white"
-                            variant="mainNavFeatured"
-                            icon="arrow-right"
-                            animate="right"
-                            className={styles.featuredCta}
-                          >
-                            {link.label}
-                          </CtaLink>
-                        </li>
-                      ))}
-                    </ul>
+                    {!!featuredLinks?.length && (
+                      <ul className={styles.linkList}>
+                        {featuredLinks.map((link) => (
+                          <li key={link._uid} className={styles.featuredItem}>
+                            <CtaLink
+                              sbLink={link.link}
```

Comment:
I previously set up CtaLink to take both SbLink directly, or a href (so it can be used with Storyblok component directly as well as hardcoded with routes). I forgot I supports SbLink since it's been a while since I use it. When you pass a SbLink directly into CtaLink, it automatically add a / in front of the cached_url which is what we need.